### ISSUE TITLE
Looking for a new owner!

### ISSUE THREAD

**Issue by kevinrenskers** [raml2html/raml2html#153]
As you may have noticed, lately I've been slow to react to issues and the new templates are *still* not implemented. The reason is that I never use RAML anymore and even when I did, raml2html in its current form works for me. I don't need all the extra features that people want, and the current templates with the modals work fine for me as well.

Since I don't actually use this project anymore, maintaining it has become a burden. I simply don't have enough free time to work (for free) on a project that I don't even use myself. Thus, I am looking for someone to take over raml2html, raml2md and raml2obj. Please get in contact with me if you actively use ram2html and would like to take over ownership; add new features, fix bugs, release new versions, answer tickets, etc etc.

Until then I'll gladly accept pull requests with bug fixes but I won't be adding or accepting new features.

**Comment by DataGreed** [comment]
@kevinrenskers BTW, did you switch to something other than RAML?

**Comment by kevinrenskers** [comment]
Our company switched to swagger. For me personally I don't use anything :) I have no API's for personal projects.

**Comment by justjacksonn** [comment]
Tell them to switch back.. RAML 1.0 is out and offers a lot more! :D

**Comment by dave-irvine** [comment]
Hmm, I wonder why? I considered API blueprint, Swagger and RAML and went with RAML.

**Comment by kevinrenskers** [comment]
Haha well the developers in my company that work on the API now have the API docs integrated in the code itself, as header docs. And they're using https://github.com/gangverk/flask-swagger to extract those docs into the swagger format.

Before this change I basically maintained the docs for the API, I used RAML, and it's why I started raml2html. Now I just don't have a need for it anymore..

**Comment by justjacksonn** [comment]
I suspect the reason why many switched to Swagger 2 was the long wait for RAML 1 and the lack of project progression due to those waiting for RAML 1. I know a few in my own company were considering the switch as well as other companies were in the process. RAML 1 offers a whopping lot more possibilities than Swagger 2 presently.

**Comment by justjacksonn** [comment]
That's a shame Kevin... anyone putting docs in the code and extracting them has it backwards. The greatest thing about RAML and Swagger is the ability to use it as a single source of truth document.. pull documentation, client code, server code, automated tests, etc all from the one file that keeps everything in sync. It's tough to change the current school of thought of most developers.

**Comment by mikestowe** [comment]
Kevin - sorry to hear that.  Happy to take on ownership until you're ready to focus on this again :)  Feel free to email me, me {at} mikestowe.com

**Comment by kevinrenskers** [comment]
Mike, thanks for the offer! I am just wondering if anybody else is interested in taking over raml2html before I make a decision. 

@dwcramer aren't you interested maybe? You did a lot of work on your own fork and have a lot of ideas and opinions that I usually completely agreed with. You are/were active in the tickets and have contributed the most code as well. You seem like the ideal person to take this project to the next level.

**Comment by dwcramer** [comment]
@kevinrenskers I'd be reluctant to take it over for fear that I wouldn't be able to respond quickly enough to pull requests and bug reports, but I'll be happy to help @mikestowe out and share anything I've done that's useful. I would like to find to make it so that my stuff is just a customization of raml2html and not a fork--forking was expedient, but obviously not a good way to go long-term.

**Comment by justjacksonn** [comment]
@all I'd be willing to help when I can.. but what I think would be great with whomever takes it over, we maybe set up a chat among some of us interested and discuss future directions. One thing I would like to see more of is how to customize the output, so that I could say render html that matches my company colors, look, etc. I know the current version has templates and the new version was changing to a new template engine. It would be great to keep this alive, get it working with 1.0, and provide some example templates on how to change colors, layout, styles (bootstrap, material design, etc). If there is the possibility of that happening, I would like to participate more as I think that would really help people come up with their own look/color/layout but get the majority of the benefits of what RAML2HTML offers.

**Comment by dave-irvine** [comment]
I'd also be happy to pitch in although I've yet to contribute anything, a maintained and working documentation generator is important to me.

I've never touched nunjucks unfortunately, I'm more comfortable with Dust.

I also wonder if trying to find someone to take all 3 projects at once is troublesome, I know I at least dont need anything to do with markdown so I'd be less willing to take that on than the html generator.

Just my £0.02

**Comment by kevinrenskers** [comment]
Nah raml2md separate, I don't "expect" anyone to take it all :) raml2obj is a different story, it's simply needed for raml2html to work.

**Comment by dave-irvine** [comment]
Yes sorry should have said that yes raml2obj is a required one!

**Comment by mikestowe** [comment]
@justjacksonn, @dwcramer, @dave-irvine - I'm going to try to focus on this heavily over the next couple of weeks  to get it upgraded for RAML 1.0, but would love your help with that, or other issues as you're able :)

**Comment by nickveenhof** [comment]
I'd be happy to accept pull request in your regards. I currently have 2 pull requests open and would be willing to respond on pull requests and provide reviews. I cannot promise I can add new features but together we certainly should be able to figure it out.

**Comment by kevinrenskers** [comment]
I've sent some invitation to the raml2html org, with the intention to give more people push access, so they can review and accept pull requests, etc.

My thinking is that all new code should go via a pull request and have a code review, but other then that, that way more people should be involved with adding code & reviewing code. The actual pushing of new versions to NPM can be left to @mikestowe and me for now, but also that can be increased to more maintainers of course.

Basically anyone that wants to be involved should get push access after their first successful PR, something like that. Let's so how it goes :)

**Comment by alexec** [comment]
+1

**Comment by kevinrenskers** [comment]
Check https://github.com/raml2html/raml2html/issues/254. There have been 8 beta's of raml2html version 4 with RAML 1.0 support.

**Comment by yamsellem** [comment]
@kevinrenskers Ah! You're in the move. Thanks so much for keeping this alive. Any idea of a official release date?

**Comment by kevinrenskers** [comment]
Soon :) Kinda depends on that array type issue which has been reported to the `datatype-expansion` project.

**Comment by yamsellem** [comment]
Perfect!

One last thing: does that mean that raml2md will also get RAML 1.0 support soon?
Thanks again.

**Comment by kevinrenskers** [comment]
Yes, there is also a beta version available for that. Once raml2obj goes out of beta, both raml2html 4 and raml2md 4 will be published too.

**Comment by yamsellem** [comment]
Is there some new templates on the way?
I can highly understand we have to wait more for that ;-).

I ask the guys of [slate](https://github.com/lord/slate) if they [may consider using RAML](https://github.com/lord/slate/issues/617) (aside of their existing custom Markdown), because their approach / layout fells great!

**Comment by kevinrenskers** [comment]
No, the new template plans are on hold for now.

**Comment by yamsellem** [comment]
I totally get that.
I hope the guys from slate will do something, things seems ready (_with the [raml2slate library](https://github.com/wdullaer/raml2slate) — [awaiting for raml2md](https://github.com/wdullaer/raml2slate/issues/1) — it should works, even if it feels a little complicated_).
